doc/api/cli.md

```diff
@@ -235,6 +235,15 @@ added: v7.7.0
 A comma separated list of categories that should be traced when trace event
 tracing is enabled using `--trace-events-enabled`.
 
+### `--trace-event-format`
```

Comment:
Some of our flags are pluralized (`trace-events`), while others are not. My vote is for pluralizing all of them. Thoughts?